lib/wanda_web/views/v2/execution_view.ex

```diff
@@ -0,0 +1,15 @@
+defmodule WandaWeb.V2.ExecutionView do
```

Comment:
where and how does this view get used? I am probably not remembering the wiring with the controller :thinking: 

Shouldn't we have in our controllers something like
```elixir
render(conn, "start.json", page: %{
          execution_id: execution_id,
          group_id: group_id
        })
```

we are doing a plain
```elixir
conn
      |> put_status(:accepted)
      |> render(
        accepted_execution: %{
          execution_id: execution_id,
          group_id: group_id
        }
      )
```

Reply:
It's a Phoenix convention, if we don't declare this module the controller explodes :sweat_smile: